src/components/NavBar.css

```diff
@@ -0,0 +1,34 @@
+.navigation-container {
+  background-color: var(--white);
+  box-shadow: 0 3px 3px -1px var(--primary-shadow);
+}
+
+.navigation-brand {
+  font-size: 2.25rem;
+  text-decoration: none;
+  color: var(--primary);
+}
+
+.navigation-bar .textbox-icon-container {
+  color: var(--primary);
+  border-color: var(--primary);
+}
+
+.navigation-bar ::placeholder {
+  color: var(--primary);
+}
+
+.navigation-bar .textbox {
+  outline: none;
+  caret-color: var(--primary);
+  color: var(--primary);
+}
+
+.navigation-login > .btn {
+  /* font-size: 1rem; */
```

Comment:
Can remove comments if not needed

Reply:
Thank you for pointing this. Will remove